### ISSUE TITLE
Line no longer required in start.sh?

### ISSUE THREAD

**Issue by RedAtRareCandy** [luizfnunes/SingleCOne-Conky#7]
https://github.com/luizfnunes/SimpleC-one-Conky/blob/7723011e98b2ffb2b0eb1c3f93d55673607b0c89/SimplecOne/start.sh#L2

The weather file is now a .sh shell script and the conkyrc1 file launches it, so could this line be removed?

**Comment by luizfnunes** [comment]
Yes, it was my mistake not to change the extension from .py to .sh. Thanks!
